docs/rules/no-ember-testing-in-module-scope.md

```diff
@@ -0,0 +1,35 @@
+## Ember.testing is not allowed in modules scope
+
+### Rule name: `no-ember-testing-in-module-scope`
+
+Since ember-cli-qunit@4.1.0 / ember-qunit@3.0.0, Ember.testing is only set to
+true while a test is executing instead of all the time. Also, Ember.testing is a
+getter/setter in Ember and destructuring will only read its value at the time
+the module is evaluated.
+https://github.com/emberjs/ember-test-helpers/pull/227
+
+```javascript
+// Good
+export default Component.extend({
+  init() {
+    this.isTesting = Ember.testing
```

Comment:
I don't think this should be recommended either. `Ember.testing` should be used directly wherever you need it (instead of this example using `if (this.isTesting) {}` somewhere).

Reply:
Should I just remove it from the docs, or should i change the rule to disallow any assignment of `Ember.testing`?

Reply:
I see this quation has been answered below